lib/broccoli/angular-broccoli-less.js

```diff
@@ -0,0 +1,57 @@
+/* jshint node: true, esversion: 6 */
+'use strict';
+
+let less;
+try {
+  less = require(`${process.env.PWD}/node_modules/less`);
```

Comment:
Because we are using the `less` from the project dir, assuming it is not installed globally. I can add another `try {} catch (e) {}` block to check if its installed globally?